src/providers/exploreState.tsx

```diff
@@ -371,10 +372,17 @@ function exploreReducer(
      * Clear all filters
      **/
     case ExploreActionKind.ClearFilters: {
+      const filterCount = 0;
+      const filterState: SelectedFilter[] = [];
       return {
         ...state,
-        filterCount: 0,
-        filterState: [],
+        entityPageState: updateEntityPageState(
```

Comment:
`updateEntityPageState` function to update the entity's `entityPageState` with the cleared filter state + count.